packages/backend/src/apps/toolbox/common/condition-is-true.ts

```diff
@@ -1,4 +1,29 @@
-import type { IJSONObject } from '@plumber/types'
+import type { IJSONObject, IJSONValue } from '@plumber/types'
+
+function compareNumbers(
+  field: IJSONValue,
+  condition: IJSONValue,
```

Comment:
Since this is a helper function, we can leverage type system to help us check inputs (as long as we do the fallthrough i suggested below)

```suggestion
  condition: 'gte' | 'gt' | 'lte' | 'lt',
```

If we do this then there is no need for `default` case.